### DISCUSSION TITLE
Multiple Pi-holes?

### DISCUSSION THREAD

**Discussion by g6nhu** [arendruni/homebridge-pihole#37] [General]
Hi,
I have two Pi-holes on my network and I dish out both IPs so that clients can use either of them for redundancy.   The plugin is great but is there any way I can use it to disable both Pi-holes?
Thanks.

**Comment by MeasurementRick** [comment]
Same issue here. I'm curious if there's a way to set the config such that it can turn off both Pi-hole instances at the same time.

**Comment by arendruni** [comment]
Hi! Sorry for the late answer. 
With the current package setup there’s no way to achieve this, but you can try to define two different Pi-holes accessories in the Homebridge config.json (see #21 on how to setup multiple accessories) and then setup a Scene in Home app to turn on/off both Pi-holes at the same time.

**Reply by g6nhu** [reply]
A perfect answer, thank you.  That's done exactly what I want.

**Reply by MeasurementRick** [reply]
Thanks for the tip about multiple accessories + creating a Scene in Apple Home. Works great!
